<launch>
 <!-- Weighing_Balance_Details -->

	<arg
		name="device_name"
		default="kern_pcb2500"/>
	<arg
		name="connection_mode"
		default="serial"/>
	<arg
		name="address"
		default="None"/>
	<arg
		name="port"
		default="/dev/ttyUSB0"/>
	<arg
		name="simulation"
		default = 'False'/>
	<node
		name="Kern_PCB2500_Driver"
		pkg="roslabware_drivers"
		type="kern_pcb2500"
		args="
			-d $(arg device_name)
			-c $(arg connection_mode)
			-add $(arg address)
			-p $(arg port)
			-l $(arg simulation)"
		output="screen"/>


 <!-- Door_Details -->

	<arg
		name="Door_name"
		default="Balance_Door"/>
	<arg
		name="Door_connection_mode"
		default="serial"/>
	<arg
		name="Door_address"
		default="None"/>
	<arg
		name="Door_port"
		default="/dev/ttyACM0"/>
	<arg
		name="Door_simulation"
		default = 'False'/>
	<node
		name="Kern_door_driver"
		pkg="roslabware_drivers"
		type="kern_door"
		args="
			-de $(arg Door_name)
			-co $(arg Door_connection_mode)
			-addr $(arg Door_address)
			-po $(arg Door_port)
			-s $(arg Door_simulation)"
		output="screen"/>
</launch>
